ngrinder-controller/src/main/java/org/ngrinder/perftest/service/ConsoleManager.java

```diff
@@ -136,7 +136,7 @@ public SingleConsole getAvailableConsole(ConsoleProperties baseConsoleProperties
 					consoleCommunicationSetting.setInactiveClientTimeOut(config.getInactiveClientTimeOut());
 				}
 				SingleConsole singleConsole = new SingleConsole(config.getCurrentIP(), consoleEntry.getPort(),
-						consoleCommunicationSetting, baseConsoleProperties);
+						consoleCommunicationSetting, baseConsoleProperties, config.getCsvSeparator());
```

Comment:
instead of set csv separator in singleconsole contructure.. it might be better to set it with separate setter.

``` singleConsole.setCsvSeparator(String csvSeparator)```